tests/CLI/CLI.h

```diff
@@ -55,4 +55,16 @@ struct CompleteIncompleteStruct;
 typedef struct IncompleteStruct IncompleteStruct;
 
 DLL_API IncompleteStruct* createIncompleteStruct();
-DLL_API void useIncompleteStruct(IncompleteStruct* a);
\ No newline at end of file
+DLL_API void useIncompleteStruct(IncompleteStruct* a);
```

Comment:
We try placing non-generator specific tests in `Common.h`. Could you please try moving it there?

Reply:
The fix is specific to CLI because it is fixing pointer dereferencing.

Reply:
@IIFE I meant that functions taking a pointer to an enum can be mapped to C# as well. So please try moving the test there as I expect no trouble from the C# gen, it can already handle such pointers.

Reply:
Just tried common test. Unfortunately the C# generation also generates pointer to by ref enum param, so moving the test to common will break unless the C# part is fixed, which is probably worth doing in another PR, so maybe keep the CLI tests?

Reply:
Thanks for checking, we can fix it later.